### ISSUE TITLE
Add a modem argument to gps.locate()

### ISSUE THREAD

**Issue by SDHKLF** [SquidDev-CC/CC-Tweaked#1585]
As far as I can tell, gps.locate cannot be used with a wireless modem connected to the computer over a wired network. Adding an argument to use a specific modem, or having gps.locate search for modems over the network would make this possible.

**Comment by SquidDev** [comment]
Can you explain what you're trying to do here? If you've got a wireless modem connected to a wired network, I'd expect it to have a constant position, and so you don't need to use GPS at all.

**Comment by SDHKLF** [comment]
Right, in this case the computer is on a create contraption, but because turtles can connect to wired networks it has more potential utility than just this instance.

**Comment by SquidDev** [comment]
Right, that makes sense, thanks!

I think my main concern is that `gps.locate` returns the position of the adjacent block, which in this set up will be the wired modem rather than the calling computer. It sounds like sometimes that is useful, but definitely something we need to document when adding this.
